charts/azure-service-operator/crds/apiextensions.k8s.io_v1beta1_customresourcedefinition_azurepublicipaddresses.azure.microsoft.com.yaml

```diff
@@ -46,9 +41,11 @@ spec:
           properties:
             idleTimeoutInMinutes:
               type: integer
+            ipTags:
+              additionalProperties:
+                type: string
+              type: object
```

Comment:
New properties on purpose?

Reply:
`ipTags` was added in #1246 - it's a `map[string]string` so I think this is right.